docs/resources/trust_token.md

```diff
@@ -0,0 +1,57 @@
+# lxd_trust_token
```

Comment:
If you could clearly convey that those `lxd_trust_token` are meant to be issued by the LXD TF provider for the benefit of other clients wanting to interact with LXD rather than be used by the LXD TF provider itself.